db_upgrade_scripts/mosip_prereg/sql/1.3.0_to_1.3.1_rollback.sql

```
@@ -0,0 +1,25 @@
+-- raw identifier can no longer be recovered, and an older (pre-1.3.1) application build — which
+-- compares the raw auth userId directly against these columns — will fail all ownership checks,
+-- silently hiding applicants' own records from them.
+--
+-- Safe to run ONLY if the identity migration has not yet populated any ownership column with a
+-- UUID (i.e. immediately after upgrade, before any login/create/book/update traffic). If migrated
+-- data exists, do NOT roll back with this script alone — the UUID columns must first be reverted
+-- to their raw values (requires decrypting identifier_encrypted from user_details BEFORE this drop).
+-- WARNING: This will permanently delete all surrogate UUID mappings stored in this table.
+-- ------------------------------------------------------------------------------------------
+
+DROP INDEX IF EXISTS prereg.idx_user_details_encrypted_dtimes;
```

SQLFluff (4.2.2)

[error] DROP statement should use CONCURRENTLY to avoid locking the table.

(PG01)